main.go

```diff
@@ -76,6 +76,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "SSP")
 		os.Exit(1)
 	}
+	if err = (&sspv1alpha1.SSP{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
Can you add an option to not enable the webhook? It will be useful when running the operator locally using `make run`. Or if it is possible to run the webhook locally, can you add some description to `README.md`, explaining how to setup certificates?

Reply:
`make run ENABLE_WEBHOOKS=false` might just work, I'll double-check and add it to the readme.

Reply:
I was actually missing a check (https://v0-19-x.sdk.operatorframework.io/docs/golang/webhooks/#update-maingo-so-that-running-locally-works)
Should be fixed now.